pixtools/responsiveness/CI_Analysis_pointplot.py

```diff
@@ -0,0 +1,81 @@
+def significance_extraction(CI):
```

Comment:
It seems that the functions in this module are duplicated elsewhere in one of your own scripts. What's the difference between them?